website/source/docs/platform/k8s/helm.html.md

```diff
@@ -340,6 +349,35 @@ to run the sync program.
       The name of the private key for the certificate file within the
       `secretName` secret.
 
+  * <a name="v-connectinject-acl-bindingrule-selector" href="#v-connectinject-acl-bindingrule-selector">`namespaceSelector`</a> (`string: "serviceaccount.name!=default"`) -
+  A [selector](/docs/acl/acl-auth-methods.html#binding-rules) for restricting automatic injection to only matching services based on
+  their associated service account. By default, services using the `default` Kubernetes service account will not have a proxy injected.
+
+  * <a name="v-connectinject-centralconfig" href="#v-connectinject-centralconfig">`centralConfig`</a> - Values that configure
+  Consul's [central configuration](/docs/agent/config_entries.html) feature (requires Consul v1.5+ and consul-k8s v0.8.1+).
+
+      - <a name="v-connectinject-centralconfig-enabled" href="#v-connectinject-centralconfig-enabled">`secretName`</a> (`string: null`) -
+      Turns on the central configuration feature. Pods that have a Connect proxy injected will have their service
+      automatically registered in this central configuration.
+
+      - <a name="v-connectinject-centralconfig-defaultprotocol" href="#v-connectinject-centralconfig-defaultprotocol">`defaultProtocol`</a> (`string: null`) -
+      If defined, this value will be used as the default protocol type for all services registered with the central configuration.
+      This can be overridden by using the
+      [protocol annotation](/docs/platform/k8s/connect.html#consul-hashicorp-com-connect-service-protocol)
+      directly on any pod spec.
+
+      - <a name="v-connectinject-centralconfig-proxydefaults" href="#v-connectinject-centralconfig-proxydefaults">`proxyDefaults`</a> (`string: "{}"`) -
+      This value is a raw json string that will be applied to all Connect proxy sidecar pods. It can include any valid configuration
+      for the configured proxy.
+
+        ```yaml
```

Comment:
Is this little code snippet documented anywhere else? I included it in the guide but don't remember talking about it too much. Since its currently required for metrics I think we should say something about it or I should add a note about it in the guide, or both.

Reply:
I included this snippet only to illustrate the necessary formatting for using this option in the Helm chart. I don't think that this is the place to do any additional explanation as it won't be very discoverable. If more info is needed, it might make more sense to add it into the Kubernetes/Connect page as another subsection.

Reply:
Sounds good